### PR TITLE
chore: ensure colab sample notebooks are tested

### PR DESCRIPTION
**Pull request by tswast**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-bigquery-dataframes/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes internal issue 321013333
 🦕

### CODE REVIEW

File: noxfile.py [hunk 1]
```diff
@@ -719,16 +733,33 @@ def notebook(session):
     pytest_command = [
         "py.test",
         "--nbmake",
-        "--nbmake-timeout=600",
+        "--nbmake-timeout=900",  # 15 minutes
```
Comment (ashleyxuu) [inline noxfile.py:736]: Is there a reason why we increase the timeout limit specifically to 15 mins. Do we test the boundary?

**Comment by tswast** [comment]
Tested locally, and the session is now passing:

```
$ nox -r -s notebook-3.9
nox > Running session notebook-3.9
nox > Re-using existing virtual environment at .nox/notebook-3-9.
nox > python -m pip install -e '.[all]'
nox > python -m pip install pytest pytest-xdist pytest-retry nbmake google-cloud-aiplatform matplotlib seaborn
nox > python /usr/local/google/home/swast/src/bigframes/scripts/notebooks_fill_params.py notebooks/regression/easy_linear_regression.ipynb notebooks/regression/sklearn_linear_regression.ipynb notebooks/dataframes/dataframe.ipynb notebooks/generative_ai/bq_dataframes_llm_kmeans.ipynb notebooks/generative_ai/large_language_models.ipynb notebooks/generative_ai/bq_dataframes_llm_code_generation.ipynb notebooks/visualization/bq_dataframes_covid_line_graphs.ipynb notebooks/remote_functions/remote_function.ipynb notebooks/getting_started/getting_started_bq_dataframes.ipynb
nox > py.test --nbmake --nbmake-timeout=900 -nauto notebooks/regression/easy_linear_regression.ipynb notebooks/regression/sklearn_linear_regression.ipynb notebooks/dataframes/dataframe.ipynb notebooks/generative_ai/bq_dataframes_llm_kmeans.ipynb notebooks/generative_ai/large_language_models.ipynb notebooks/generative_ai/bq_dataframes_llm_code_generation.ipynb notebooks/visualization/bq_dataframes_covid_line_graphs.ipynb notebooks/remote_functions/remote_function.ipynb notebooks/getting_started/getting_started_bq_dataframes.ipynb
============================================================ test session starts ============================================================
platform linux -- Python 3.9.16, pytest-7.4.4, pluggy-1.4.0
rootdir: /usr/local/google/home/swast/src/bigframes
configfile: pytest.ini
plugins: retry-1.6.1, xdist-3.5.0, mock-3.12.0, nbmake-1.4.6
12 workers [9 items]      
.........                                                                                                                             [100%]
======================================================= 9 passed in 751.43s (0:12:31) =======================================================
nox > python /usr/local/google/home/swast/src/bigframes/scripts/notebooks_restore_from_backup.py notebooks/regression/easy_linear_regression.ipynb notebooks/regression/sklearn_linear_regression.ipynb notebooks/dataframes/dataframe.ipynb notebooks/generative_ai/bq_dataframes_llm_kmeans.ipynb notebooks/generative_ai/large_language_models.ipynb notebooks/generative_ai/bq_dataframes_llm_code_generation.ipynb notebooks/visualization/bq_dataframes_covid_line_graphs.ipynb notebooks/remote_functions/remote_function.ipynb notebooks/getting_started/getting_started_bq_dataframes.ipynb
nox > py.test --nbmake --nbmake-timeout=900 notebooks/location/regionalized.ipynb
nox > py.test --nbmake --nbmake-timeout=900 notebooks/location/regionalized.ipynb
nox > py.test --nbmake --nbmake-timeout=900 notebooks/location/regionalized.ipynb
nox > py.test --nbmake --nbmake-timeout=900 notebooks/location/regionalized.ipynb
nox > py.test --nbmake --nbmake-timeout=900 notebooks/location/regionalized.ipynb
nox > py.test --nbmake --nbmake-timeout=900 notebooks/location/regionalized.ipynb
============================================================ test session starts ============================================================
platform linux -- Python 3.9.16, pytest-7.4.4, pluggy-1.4.0
rootdir: /usr/local/google/home/swast/src/bigframes
configfile: pytest.ini
plugins: retry-1.6.1, xdist-3.5.0, mock-3.12.0, nbmake-1.4.6
collected 1 item                                                                                                                            

notebooks/location/regionalized.ipynb ============================================================ test session starts ============================================================
platform linux -- Python 3.9.16, pytest-7.4.4, pluggy-1.4.0
rootdir: /usr/local/google/home/swast/src/bigframes
configfile: pytest.ini
plugins: retry-1.6.1, xdist-3.5.0, mock-3.12.0, nbmake-1.4.6
collecting ... ============================================================ test session starts ============================================================
platform linux -- Python 3.9.16, pytest-7.4.4, pluggy-1.4.0
collected 1 item                                                                                                                            

notebooks/location/regionalized.ipynb ============================================================ test session starts ============================================================
platform linux -- Python 3.9.16, pytest-7.4.4, pluggy-1.4.0
rootdir: /usr/local/google/home/swast/src/bigframes
configfile: pytest.ini
plugins: retry-1.6.1, xdist-3.5.0, mock-3.12.0, nbmake-1.4.6
collecting ... rootdir: /usr/local/google/home/swast/src/bigframes
configfile: pytest.ini
plugins: retry-1.6.1, xdist-3.5.0, mock-3.12.0, nbmake-1.4.6
collecting ... ============================================================ test session starts ============================================================
platform linux -- Python 3.9.16, pytest-7.4.4, pluggy-1.4.0
collected 1 item                                                                                                                            

notebooks/location/regionalized.ipynb rootdir: /usr/local/google/home/swast/src/bigframes
configfile: pytest.ini
plugins: retry-1.6.1, xdist-3.5.0, mock-3.12.0, nbmake-1.4.6
collected 1 item                                                                                                                            

collected 1 item                                                                                                                            

notebooks/location/regionalized.ipynb ============================================================ test session starts ============================================================
platform linux -- Python 3.9.16, pytest-7.4.4, pluggy-1.4.0
rootdir: /usr/local/google/home/swast/src/bigframes
configfile: pytest.ini
plugins: retry-1.6.1, xdist-3.5.0, mock-3.12.0, nbmake-1.4.6
collected 1 item                                                                                                                            

notebooks/location/regionalized.ipynb .                                                                                               [100%]

======================================================= 1 passed in 122.27s (0:02:02) =======================================================
.                                                                                               [100%]

======================================================= 1 passed in 132.72s (0:02:12) =======================================================
.                                                                                               [100%]

======================================================= 1 passed in 135.59s (0:02:15) =======================================================
.                                                                                               [100%]

======================================================= 1 passed in 139.97s (0:02:19) =======================================================
.                                                                                               [100%]

======================================================= 1 passed in 147.41s (0:02:27) =======================================================
.                                                                                               [100%]

======================================================= 1 passed in 189.72s (0:03:09) =======================================================
nox > Session notebook-3.9 was successful.
```

File: noxfile.py [hunk 2]
```diff
@@ -665,19 +679,19 @@ def notebook(session):
...
+        # With the notebooks_fill_params.py script, we are able to find and
+        # replace the PROJECT_ID parameter, but not the others.
+        #
         # TODO(ashleyxu): Test these notebooks by replacing parameters with
         # appropriate values and omitting cleanup logic that may break
         # our test infrastructure.
-        "notebooks/getting_started/getting_started_bq_dataframes.ipynb",
-        "notebooks/getting_started/ml_fundamentals_bq_dataframes.ipynb",
-        "notebooks/generative_ai/bq_dataframes_llm_code_generation.ipynb",
-        "notebooks/generative_ai/bq_dataframes_llm_kmeans.ipynb",
```
Comment (ashleyxuu) [inline noxfile.py:674]: Do we still need DATASET when running `bf.read_gbq()`?

File: noxfile.py (hunk 1 above)
Comment (tswast) [reply noxfile.py:736]: The default was too slow for the LLM sample when I tested locally.

File: noxfile.py (hunk 2 above)
Comment (tswast) [reply noxfile.py:674]: I don't know what you mean. DATASET variable doesn't appear in https://github.com/googleapis/python-bigquery-dataframes/blob/677f0146acf19def88fddbeb0527a078458948ae/notebooks/generative_ai/bq_dataframes_llm_kmeans.ipynb

Comment (ashleyxuu) [reply noxfile.py:674]: We read it here, right: https://screenshot.googleplex.com/32s3pbtqnWZakiM

Comment (tswast) [reply noxfile.py:674]: That's a read, which doesn't need to be parametrized. DATASET is used a a destination in the other samples.

**Review by ashleyxuu — APPROVED** [review]

**Comment by tswast** [comment]
Looks like `bq_dataframes_llm_code_generation` had a failure. Trying a change that will hopefully help regarding the GCS bucket that sample creates.

**Comment by tswast** [comment]
Still have a failing notebook test after https://github.com/googleapis/python-bigquery-dataframes/pull/351/commits/dfee838b5c41ed6df51861a023454c4b1bbfc689 I'll maybe revert that commit and exclude that particular notebook for now.

**Pull request merged**